src/katgpucbf/utils.py

```diff
@@ -184,6 +186,76 @@ def cancel(self) -> None:
             sensor.detach(self)
 
 
+def _esterror_status(value: float) -> aiokatcp.Sensor.Status:
+    # These thresholds are thumb-sucks.
+    if 0.0 <= value < 1e-3:  # Arbitrary threshold
+        return aiokatcp.Sensor.Status.NOMINAL
+    elif value < 5e-3:  # CBF-REQ-0203 specifies 5ms
+        return aiokatcp.Sensor.Status.WARN
+    else:
+        return aiokatcp.Sensor.Status.ERROR
+
+
+def _maxerror_status(value: float) -> aiokatcp.Sensor.Status:
+    # maxerror is an over-estimate (it makes very conservative assumptions
+    # about network asymmetry and skew in the clock source). Use conservative
+    # thresholds to avoid warnings when there isn't a stratum 1 time source
+    # in the same data centre. Experimentally, maximum error is < 10ms
+    # when synchronising between Cape Town and the Karoo.
```

Comment:
Appreciate the explanation, but perhaps prefix this with `NOTE:` ?

Reply:
Not sure I follow. Isn't every comment a note about how the code works?

Reply:
Touché... But I do mean more in terms of these longer comments/reasoning. I see it's almost exclusively used in `xbgpu/engine.py`, if that helps show where my POV originates.

Again, fair point, so I concede the addition is not the most necessary change.